extras/servdata_consumer.py

```diff
@@ -42,6 +42,8 @@ def __init__(self, src):
             if resp.status_code > 304 or not resp.ok:
                 exit_with_error("Fetch failed from %s" % src)
             self.rawdata = resp.text
+        elif self.src == "-":
```

Comment:
This was probably out of scope and could have been merged separately.

Reply:
Yes, agree with that - but I did not want to end up with too many (long-outstanding) branches and decided there would be no harm in pushing this change in here....